frontend_tests/puppeteer_tests/settings.ts

```diff
@@ -382,11 +398,12 @@ async function settings_tests(page: Page): Promise<void> {
     await common.log_in(page);
     await open_settings(page);
     await test_change_full_name(page);
-    await test_get_api_key(page);
     await test_alert_words_section(page);
     await test_your_bots_section(page);
     await test_default_language_setting(page);
     await test_notifications_section(page);
+    await test_get_api_key(page);
+    await test_change_password(page);
```

Comment:
We should probably add a loud comment below `test_change_password` like the following:

~~~
// IMPORTANT NOTE!  Do not add any other tests below test_change_password, as we have some known issues with our event-fetching code racing with password updates that are beyond the scope of this test.
~~~

I would also update your commit message to mention that we think it's unlikely for `settings.ts` to affect **subsequent** tests due to how we reset the test database.